exosphere-shared/src/docker-helper.ls

```diff
@@ -9,12 +9,18 @@ class DockerHelper
     child_process.exec-sync('docker ps -a --format {{.Names}}', 'utf8') |> (.includes container)
 
 
+  @ensure-container-is-running = (container-name, image) ->
+    return if child_process.exec-sync('docker ps --format {{.Names}}/{{.Status}}' , 'utf8') |> (.includes "#{container-name}/Up")
```

Comment:
This method still implements more than one thing: checking whether a docker image is running, starting a docker image, and doing its actual business logic (what its name says). 

Please extract a `container-is-running` method and a `start-container` method. Then use pattern matching to format this method more readable:

```ls
@ensure-container-is-running = (container-name, image) ->
  | @container-is-running container-name  =>  return
  | @container-exists container-name      =>  @start-container container-name
  | otherwise                             =>  @run-image container-name, image
```

This format reads almost like plain English, and tells exactly what this method is doing!